src/defaults.mligo

```diff
@@ -46,8 +46,8 @@ let default_config (data : initial_config_data) : config =
     proposal_expired_level = data.proposal_expired_level;
   }
 
-
-let default_storage (data, config_data : initial_storage_data * initial_config_data ) : storage =
+let default_storage (initial_data: initial_data) : storage =
+  let (data, config_data) = (initial_data.storage_data, initial_data.config_data) in
```

Comment:
This distinction seems quite arbitrary now, as well as the definition of `initial_data` itself.

What do you think of making a single `initial_data` and merge the fields of `initial_config_data` and `initial_storage_data`?

Or perhaps, to match it even more, maybe remove the `storage_data` field from the `initial_data` type and move all the fields of `initial_storage_data` into `initial_data` :thinking:

Reply:
Have changed by moving fields in `inital_storage_data` to `inital_data`.